src/System.Private.ServiceModel/tools/SelfHostedWcfService/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Administration" version="7.0.0.0" targetFramework="net451" userInstalled="true" />
```

Comment:
I still saw `Microsoft.Web.Administration` in the merged code. Please open issue to clean this up. Since this is the only package dependency we have, this will eliminate everything related to package restore too.